### DISCUSSION TITLE
can you add jwt/oauth/cookies/... authentication  in vs

### DISCUSSION THREAD

**Discussion by ilivecoding** [dotnet/aspnetcore#49815] [Ideas]
![图片](https://github.com/dotnet/aspnetcore/assets/7898334/595dd737-c1c3-47b3-86a8-14d245ac4964)

**Comment by martincostello** [comment]
Could you provide some more detail as well as some text, preferably in English?

With a screenshot people can't easily even copy the text in question to try running it through a translator to get some more context on what you're asking.

**Reply by ilivecoding** [reply]
if i install jtw/oauth npm package, then my webapplication will add related code automatic in my project.
